lib/src/pylons_wallet_comm_util.dart

```diff
@@ -192,27 +190,23 @@ class PylonsWalletCommUtil {
     }
   }
 
-  /// This method checks whether the response contains the error of the supplied type or not
-  /// [Input] : [err] is the type of error , [response]  is the response in which the error is to be check,
-  /// [exception] is the exception which will be thrown if supplied error found
+  /// Checks whether [response] contains the provided error [err].
+  /// If found, throws [exception].
   ///
-  /// Can throw  the following exception in the event of no matching key in response
-  /// or having error related with this key
-  ///
-  /// [exception] : UnhandledErrorException if something wrong happens
-  static bool procError(
+  /// Alternatively, throws an [UnhandledErrorException] if an error other than
+  /// that expected is found.
+  static void checkError(
       String err, Tuple2<String, List<String>> response, Exception exception) {
     if (response.value2[0] == err) {
-      try {
-        throw exception;
-      } catch (e) {
-        throw UnhandledErrorException(
-            err, 'Bad error passed: ${response.value2}');
-      }
+      throw exception;
+    } else {
+      throw UnhandledErrorException(err, 'Bad error passed: ${response.value2}');
     }
-    return false;
   }
 
+  /// Parses the base64-encoded UTF8 string [response] and outputs a [Tuple2<String, List<String>>].
+  /// The first element of the tuple is the response's key; the second element is a list containing
+  /// the response data.
   static Tuple2<String, List<String>> procResponse(String response) {
     var decoder = const Utf8Decoder();
     var splut = response.split(',');
```

Comment:
Let's use a more meaningful variable other than "splut".
Outside this fix, this is good clean code.

Reply:
Is it not meaningful? Once it's been split, it's now splut. But perhaps too much of a pun.